### PR TITLE
Feature/form component

### PR DESCRIPTION
**Pull request by SyleimanovSergey**

### CODE REVIEW

File: src/utils/submitForms.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+export const submitFormIssues = (
+	values: FieldIssues,
+	actions: FormikHelpers<FieldIssues>
+): void => {
+	setTimeout(() => {
+		console.log(values);
+		actions.setSubmitting(false);
+	}, 1000);
+};
```
Comment (Polina04) [inline src/utils/submitForms.ts]: I hope this is just a template
submit functions are not `utils`, please create a custom hooks for them

File: src/interfaces/commonForm.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { ChangeEvent } from 'react';
```
Comment (Polina04) [inline src/interfaces/commonForm.ts]: why do you have two imports from react in one file?
Does the eslint work correctly?

File: src/components/Forms/IssueForm/index.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import {
+	InterfaceOptions,
+	TypeInputFormikControl
+} from 'interfaces/commonForm';
+import { initialIssues } from 'utils/initialValuesForms';
+import { submitFormIssues } from 'utils/submitForms';
+import styles from './index.module.scss';
+
+const IssueForm: React.FC = () => {
+	const optionsPriority: InterfaceOptions[] = [
```
Comment (Polina04) [inline src/components/Forms/IssueForm/index.tsx]: move it above your component

File: src/components/common/Form/TextError/index.tsx [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import styles from './index.module.scss';
+
+const TextError = ({ ...props }): JSX.Element => {
```
Comment (Polina04) [inline src/components/common/Form/TextError/index.tsx]: redundant props spreading

**Review by Polina04 — COMMENTED** [review]
- please check imports order and empty lines between imports groups in all your new files
 - please add empty lines before `return`s

**Review by DmitriyHnatiuk — APPROVED** [review]

**Review by NiFroPP — APPROVED** [review]

File: src/components/Forms/RegistrationForm/index.tsx [hunk 5]
```diff
@@ -1,18 +1,19 @@
 import React from 'react';
 import { Form, Formik } from 'formik';
 
-import { TypeInputFormikControl } from 'interfaces/commonForm';
 import FormikControl from 'components/common/Form/FormikControl';
 
 import { initialValuesRegistration } from 'utils/initialValuesForms';
-import { submitFormRegistration } from 'utils/submitForms';
+import { TypeInputFormikControl } from 'interfaces/commonForm';
+import { useSubmitFormRegistration } from '../../../hooks/submitForms';
```
Comment (Polina04) [inline src/components/Forms/RegistrationForm/index.tsx]: '../../../hooks'?
`hooks/submitForms`

**Review by Polina04 — APPROVED** [review]

**Pull request merged**